dev/update-dep-lock.sh

```diff
@@ -1,19 +1,61 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Updated because the new logic uses Bash arrays to deal (more) sanely with lists of arguments, etc.